core/src/components/action-sheet/action-sheet.tsx

```diff
@@ -386,7 +386,13 @@ export class ActionSheet implements ComponentInterface, OverlayInterface {
                 </div>
               )}
               {buttons.map((b) => (
-                <button type="button" id={b.id} class={buttonClass(b)} onClick={() => this.buttonClick(b)}>
+                <button
+                  {...b.htmlAttributes}
```

Comment:
The design document called for this to be added like the following:

```
  {...(b.htmlAttributes as any)}
```

I couldn't see any reason that it needs to be cast as `any` though - no failing linters or builds. Anyone know if this is needed for some other reason?

Reply:
I'm not sure. I even checked if Visual Studio would give an error of sorts but nothing came up.